frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -156,6 +181,7 @@ type LocalVolumeDiscoveryResult = K8sResourceCommon & {
       };
       type: string;
       vendor: string;
+      node?: string;
```

Comment:
`nodeName: string;` is already present in the spec.
The type `LocalVolumeDiscoveryResult` matches the CR `kind`, so we must not add any other item here.